docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -15,50 +15,24 @@ We will start with a single cluster and move to multiple clusters to illustrate
+* 
+* 
+* Export env-vars with the keys listed below. Fill in your own values as appropriate. You will need access to a domain or subdomain in Route 53 in AWS:
 
-  ```bash
-  AWS_DNS_PUBLIC_ZONE_ID=Z01234567US0IQE3YLO00
-  ZONE_ROOT_DOMAIN=jbloggs.hcpapps.net
-  LOG_LEVEL=1
-  ```
-
-  | Env Var                  | Example Value           | Description                                           |
-  |--------------------------|-------------------------|-------------------------------------------------------|
-  | `ZONE_ROOT_DOMAIN`       | `jbloggs.hcpapps.net`   | Hostname for the root Domain                          |
-  | `AWS_DNS_PUBLIC_ZONE_ID` | `Z01234567US0IQE3YLO00` | AWS Route 53 Zone ID for specified `ZONE_ROOT_DOMAIN` |
-  | `LOG_LEVEL`              | `1`                     | Log level for the Controller                          |
-
-* Setup a `./aws-credentials.env` with credentials to access route 53
-
-  For example:
-
-    ```bash
-    AWS_ACCESS_KEY_ID=<access_key_id>
-    AWS_SECRET_ACCESS_KEY=<secret_access_key>
-    AWS_REGION=eu-west-1
-    ```
-
-  | Env Var                 | Example Value          | Description                                                    |
-  |-------------------------|------------------------|----------------------------------------------------------------|
-  | `AWS_ACCESS_KEY_ID`     | `AKIA1234567890000000` | Access Key ID, with access to resources in Route 53            |
-  | `AWS_SECRET_ACCESS_KEY` | `Z01234567US0000000`   | Access Secret Access Key, with access to resources in Route 53 |
-  | `AWS_REGION`            | `eu-west-1`            | AWS Region                                                     |
-
-
-* We're going to use an environment variable, `MGC_SUB_DOMAIN`, throughout this walkthrough. Simply run the below in each window you create:
-
-  For example:
-  ```bash
-  export MGC_SUB_DOMAIN=myapp.jbloggs.hcpapps.net
-  ```
+ | Env Var                      | Example Value               | Description                                                    |
+|------------------------------|-----------------------------|----------------------------------------------------------------|
+  | `MGC_ZONE_ROOT_DOMAIN`       | `jbloggs.hcpapps.net`       | Hostname for the root Domain                                   |
+  | `MGC_AWS_DNS_PUBLIC_ZONE_ID` | `Z01234567US0IQE3YLO00`     | AWS Route 53 Zone ID for specified `MGC_ZONE_ROOT_DOMAIN`      | | 
+  | `MGC_AWS_ACCESS_KEY_ID`      | `AKIA1234567890000000`      | Access Key ID, with access to resources in Route 53            |
+  | `MGC_AWS_SECRET_ACCESS_KEY`  | `Z01234567US0000000`        | Access Secret Access Key, with access to resources in Route 53 |
+  | `MGC_AWS_REGION`             | `eu-west-1`                 | AWS Region                                                     |
+  | `MGC_SUB_DOMAIN`             | `myapp.jbloggs.hcpapps.net` | AWS Region                                                     |
```

Comment:
I wonder should this section be moved to after the explanation of the use of two terminal windows?

Reply:
I added a note under the terminal sessions section which hopefully addresses this which states that MGC_SUB_DOMAIN is required in both terminals.